### DISCUSSION TITLE
Number of CPUs in the benchmarking hardware

### DISCUSSION THREAD

**Discussion by dvsaisurya** [mlcommons/algorithmic-efficiency#632] [General]
What is the specification of the CPUs in benchmarking hardware? is it acceptable to offload computation to CPUs?

**Comment by priyakasimbeg** [comment]
We use [N1 machines](https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines) on GCP with 64 vCPUs and 240 GB memory. Yes, I believe offloading computation to CPUs should be acceptable, as long as your submission doesn't OOM.
